rosdep/base.yaml

```
@@ -7424,6 +7424,21 @@ qttools5-dev-tools:
   openembedded: [qttools@meta-qt5]
   rhel: [qt5-qttools-devel]
   ubuntu: [qttools5-dev-tools]
+qtwebengine5-dev:
+  alpine: null
+  arch: null
+  debian: [qtwebengine5-dev]
+  fedora: [qt5-qtwebengine]
+  gentoo: [dev-qt/qtwebengine-5.15.11_p20231120]
+  nixos: [libsForQt5.qt5.qtwebengine]
+  opensuse: null
+  osx: null
+  rhel: qt5-qtwebengine-dev
```

This needs to be a list:

```suggestion
  rhel: [qt5-qtwebengine-devel]
```